### DISCUSSION TITLE
Per-world Tickrate

### DISCUSSION THREAD

**Discussion by B1ue-Bandit** [PaperMC/Paper#10605] [Ideas]
### Is your feature request related to a problem?

Not that I know of.

### Describe the solution you'd like.

I would like the ability to set the tickrate for a particular world on a server versus a global tickrate. This would enable different worlds to run at different speeds.

### Describe alternatives you've considered.

I can't think of any alternatives other than just running multiple servers instead of multiple worlds on a single server.

**Comment by electronicboy** [comment]
The server does not work in a manner that permits this, and I generally do not see the inclination to pull off such an invasive patch, especially pre hard-fork
